go.mod

```diff
@@ -57,3 +57,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/hyperledger.org/firefly-common => ../firefly-common
```

Comment:
Should we check this in, or just have it locally?

Reply:
Probably just locally. I can remove this. I wasn't sure how checks in GitHub would work while we're in such a state of flux - though obviously this particular line will never work there, so I'll remove it.